uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -128,12 +132,12 @@ var Promise = require('es6-promise').Promise;
    *
    * @return {Array}
    */
-  function scenariosList() {
+  function scenariosList () {
     var scenariosPath = backstopDir + 'scenarios/';
 
     return _(fs.readdirSync(scenariosPath))
       .filter(function (scenario) {
-        return argv.configFile ? scenario === argv.configFile : true;
+        return argv.configFile ? scenario === argv.configFile : true && scenario.endsWith('.json');
```

Comment:
why is this additional check necessary?

Reply:
If you have a .DS_Store file or anything else other than a `.json` file in the folder, the code tries to read that file too and the tasks stop without any error. This is prevent that. 
Sorry, Forgot to mention this in the doc, updating.